package.json

```diff
@@ -93,6 +93,7 @@
     "ampersand-events": "^2.0.2",
     "babel-polyfill": "^6.26.0",
     "bowser": "^1.9.4",
+    "builtin-modules": "^3.1.0",
```

Comment:
What is this for?

Reply:
for some reason, [this](https://github.com/webex/react-widgets/blob/master/scripts/utils/deps.js#L6) started failing whenever I tried to build

Reply:
this was an issue with wiping `node_modules` and `package-lock` and "starting fresh" with a `npm i`
Reverting back to the `package-lock` in the master branch and running `npm ci` corrected the issue(s)

Reply:
Gotcha, if it is required, let's separate it into another commit with the message `fix(package): add builtin-modules package`.